src/sentry/api/endpoints/notifications/notification_actions_index.py

```diff
@@ -21,9 +22,21 @@
 class NotificationActionsPermission(OrganizationPermission):
     scope_map = {
         "GET": ["org:read", "org:write", "org:admin"],
-        "POST": ["org:write", "org:admin"],
-        "PUT": ["org:write", "org:admin"],
-        "DELETE": ["org:write", "org:admin"],
+        "POST": [
+            "org:read",
+            "org:write",
+            "org:admin",
+        ],
+        "PUT": [
+            "org:read",
+            "org:write",
+            "org:admin",
+        ],
+        "DELETE": [
+            "org:read",
+            "org:write",
+            "org:admin",
+        ],
```

Comment:
this is an organization endpoint because you can pass multiple projects to attach the same notification action to them. we have to do some filtering to make sure the member has the correct access for those projects